docs/images/TrajectoryPlanner.svg

```diff
@@ -0,0 +1,3 @@
; margin-left: 1201px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">Convert Position-based Trajectory to Velocity Based</div></div></div></foreignObject><text x="1260" y="114" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">Convert Position-bas...</text></switch></g><path d="M 1510 110 L 1853.63 110" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 1858.88 110 L 1851.88 113.5 L 1853.63 110 L 1851.88 106.5 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 101px; margin-left: 1671px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 11px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; background-color: #ffffff; white-space: nowrap; ">Complete Trajectory to wheel Matrix and controller</div></div></div></foreignObject><text x="1671" y="104" fill="#000000" font-family="Helvetica" font-size="11px" text-anchor="middle">Complete Trajectory to wheel Matrix and controller</text></switch></g><rect x="1390" y="80" width="120" height="60" rx="9" ry="9" fill="#ffffff" stroke="#000000" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 110px; margin-left: 1391px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">Linear Interpolation of Constant Time segments</div></div></div></foreignObject><text x="1450" y="114" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">Linear Interpolation...</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://desk.draw.io/support/solutions/articles/16000042487" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
Could we add a side-note about what `Balance Segment Duration` means? That really confused me until you explained it.

Reply:
Added 👍